terraform/app/modules/paas/main.tf

```diff
@@ -31,7 +32,29 @@ resource cloudfoundry_app web_app {
 }
 
 resource cloudfoundry_route web_app_route {
-  domain   = data.cloudfoundry_domain.cloudapps_digital.id
-  space    = data.cloudfoundry_space.space.id
+  domain = data.cloudfoundry_domain.cloudapps_digital.id
+  space = data.cloudfoundry_space.space.id
```

Comment:
No change here, but I think we should start running `terraform fmt` to make this consistent